minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/status/reporters/StatusLoggerTest.java

```diff
@@ -142,10 +144,7 @@ public void testError() {
     // Exception testing
```

Comment:
I know it's not your code, but please remove the comment, as it does not hold any additional or valuable information